<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformComp</class>
 <widget class="QGroupBox" name="TransformComp">
  <property name="windowTitle">
   <string>Transform</string>
  </property>
  <property name="title">
   <string>Transform</string>
  </property>
  <layout class="QFormLayout" name="transformLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="positionLabel">
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="positionLayout">
     <item>
      <widget class="ValuePicker" name="positionX">
       <property name="minimum">
        <double>-340282346638528859811704183484516925440.000000000000000</double>
       </property>
       <property name="maximum">
        <double>340282346638528859811704183484516925440.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValuePicker" name="positionY">
       <property name="minimum">
        <double>-340282346638528859811704183484516925440.000000000000000</double>
       </property>
       <property name="maximum">
        <double>340282346638528859811704183484516925440.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValuePicker" name="positionZ">
       <property name="minimum">
        <double>-340282346638528859811704183484516925440.000000000000000</double>
       </property>
       <property name="maximum">
        <double>340282346638528859811704183484516925440.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="scaleLabel">
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="scaleLayout">
     <item>
      <widget class="ValuePicker" name="scaleX">
       <property name="minimum">
        <double>-340282346638528859811704183484516925440.000000000000000</double>
       </property>
       <property name="maximum">
        <double>340282346638528859811704183484516925440.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValuePicker" name="scaleY">
       <property name="minimum">
        <double>-340282346638528859811704183484516925440.000000000000000</double>
       </property>
       <property name="maximum">
        <double>340282346638528859811704183484516925440.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValuePicker" name="scaleZ">
       <property name="minimum">
        <double>-340282346638528859811704183484516925440.000000000000000</double>
       </property>
       <property name="maximum">
        <double>340282346638528859811704183484516925440.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValuePicker</class>
   <extends>QDoubleSpinBox</extends>
   <header>RaZor/Interface/ValuePicker.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
